core/crypto/crypto.h

```diff
@@ -42,10 +42,10 @@ class CryptoKey : public Resource {
 
 protected:
 	static void _bind_methods();
-	static CryptoKey *(*_create)();
+	static CryptoKey *(*_create)(bool p_notify_postinitialize);
 
 public:
-	static CryptoKey *create();
+	static CryptoKey *create(bool p_notify_postinitialize);
```

Comment:
I think the diff could be made a little smaller by giving the new argument a default:

```suggestion
	static CryptoKey *create(bool p_notify_postinitialize = true);
```

Reply:
Thanks! The updates look great to me :-)